expressServer/scripts/generateUsernames.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose');
+const { uniqueNamesGenerator, adjectives, colors, animals } = require('unique-names-generator');
+const dotenv = require('dotenv');
+
+dotenv.config();
+
+mongoose.connect(process.env.MONGODB_URI, { useNewUrlParser: true, useUnifiedTopology: true })
```

Comment:
This is redundant, this code exists also in the index.js file. I would make a database lib for it, a database lib is a file that contains the functionalities related to database initialization and access and to avoid code repetitions.